### ISSUE TITLE
Overwrite Flag Not Working

### ISSUE THREAD

**Issue by endercrest** [LironEr/cypress-mochawesome-reporter#36]
**Describe the bug**
`overwrite` flag is ignored since the output directory is cleared before every run

**Expected behavior**
Expect results to continue to be added to the results directory without being deleted first.

**Environment**
`cypress-mochawesome-reporter` version: `2.1.1`
`cypress` version: `7.5.0`

Please run this command inside your project and paste its contents here (it automatically copies to your clipboard)
`npx envinfo --system --binaries --markdown | npx clipboard-cli`
Need to install the following packages:
  envinfo
Ok to proceed? (y) 
## System:
 - OS: macOS 11.2.2
 - CPU: (8) x64 Intel(R) Core(TM) i5-1038NG7 CPU @ 2.00GHz
 - Memory: 977.00 MB / 16.00 GB
 - Shell: 5.8 - /bin/zsh
## Binaries:
 - Node: 15.10.0 - ~/.nvm/versions/node/v15.10.0/bin/node
 - Yarn: 2.4.1 - /usr/local/bin/yarn
 - npm: 7.5.3 - ~/.nvm/versions/node/v15.10.0/bin/npm
 - Watchman: 4.9.0 - /usr/local/bin/watchman

**Additional context**

**Comment by LironEr** [comment]
please add your `cypress.json`

also please describe the use case, currently I'm deleting the report folder to have a clean HTML report, without outdated data

**Comment by verydevuser** [comment]
@LironEr I am having this issue too 

here is my `cypress.json`

```
{
  "component": {
    "testFiles": "**/*.test.{js,ts,jsx,tsx}",
    "componentFolder": "src"
  },
  "reporter": "cypress-mochawesome-reporter",
  "reporterOptions": {
    "reportDir": "results",
    "overwrite": false,
    "saveJson": true,
    "charts": true,
    "embeddedScreenshots": true,
    "inlineAssets": false,
    "saveAllAttempts": false
  },
  "video": false
}
```

on version `"cypress-mochawesome-reporter": "^2.3.0",`

**Comment by LironEr** [comment]
Fixed in [v2.4.0](https://github.com/LironEr/cypress-mochawesome-reporter/releases/tag/v2.4.0)

**Comment by muirdok** [comment]
btw.
issue is actual for:
`reporter: 'cypress-mochawesome-reporter',
  reporterOptions: {
    "reportDir": "results",
    "overwrite": true,
    "saveJson": true,
    "charts": true,
    "embeddedScreenshots": true,
    "inlineAssets": false,
    "saveAllAttempts": false
  },`
and 
`        "cypress": "^10.8.0",
        "cypress-mochawesome-reporter": "^3.2.2",
        "mochawesome-merge": "^4.2.1"`
